upup/pkg/fi/cloudup/networking.go

```diff
@@ -18,12 +18,14 @@ package cloudup
 
 import (
```

Comment:
Refactoring the code to use URLs rather than strings.  We can actually remove this code soon because I think we have SHA files now in google storage.